src/boot/bootctl-status.c

```diff
@@ -59,7 +60,8 @@ static int status_entries(
                 const char *esp_path,
                 sd_id128_t esp_partition_uuid,
                 const char *xbootldr_path,
-                sd_id128_t xbootldr_partition_uuid) {
+                sd_id128_t xbootldr_partition_uuid,
+                JsonVariant **v) {
```

Comment:
This is a return only param. We have this rule to name those `ret_xyz`. See https://systemd.io/CODING_STYLE/#code-organization-and-semantics for details.

hence rename v → ret_variant or so.

Reply:
Also, please set up a local variable `v` first, where you put together the object, then move it to `ret_variant` once you are done with TAKE_PTR().